libs/experimental/langchain_experimental/utilities/python.py

```diff
@@ -22,6 +23,22 @@ class PythonREPL(BaseModel):
     globals: Optional[Dict] = Field(default_factory=dict, alias="_globals")
     locals: Optional[Dict] = Field(default_factory=dict, alias="_locals")
 
+    @staticmethod
+    def sanitize_input(query: str) -> str:
```

Comment:
could we add some unit tests for this

Reply:
> could we add some unit tests for this

Hello, I have added a test example for it (the local test passed), but I keep reporting errors when I format and detect the local code
ModuleNotFoundError: No module named 'rapidfuzz'
make: *** [format_diff] Error 1
This is a problem caused by my own environment and has nothing to do with the committed code, but I don't know how to fix it @baskaryan